hs_swat_modelinstance/forms.py

```diff
@@ -0,0 +1,341 @@
+__author__ = 'Mohamed Morsy'
+from django.forms import ModelForm
+from django import forms
+from crispy_forms.layout import *
+from crispy_forms.bootstrap import *
+from models import *
+from hs_core.forms import BaseFormHelper
+
+#SWATmodelParameters element forms
+class ModelOutputFormHelper(BaseFormHelper):
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None,  *args, **kwargs):
+
+        # the order in which the model fields are listed for the FieldSet is the order these fields will be displayed
+        field_width = 'form-control input-sm'
+        layout = Layout(
+                        Field('includes_output', css_class=field_width),
+                 )
+        kwargs['element_name_label'] = 'Includes output files?'
+        super(ModelOutputFormHelper, self).__init__(allow_edit, res_short_id, element_id, element_name, layout,  *args, **kwargs)
+
+
+class ModelOutputForm(ModelForm):
+    includes_output = forms.TypedChoiceField(choices=((True, 'Yes'), (False, 'No')), widget=forms.RadioSelect)
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, *args, **kwargs):
+        super(ModelOutputForm, self).__init__(*args, **kwargs)
+        self.helper = ModelOutputFormHelper(allow_edit, res_short_id, element_id, element_name='ModelOutput')
+        self.fields['includes_output'].widget.attrs['style'] = "width:auto;margin-top:-5px"
+
+    class Meta:
+        model = ModelOutput
+        fields = ['includes_output']
+        exclude = ['content_object']
```

Comment:
@ironfroggy when I convert  fields = ['includes_output'] to  fields = ('includes_output') or  fields = tuple ['includes_output'], I get an exited error with code 1

 File "/home/docker/hydroshare/hs_swat_modelinstance/forms.py", line 34, in Meta
hydroshare_1    |     fields = tuple['includes_output']
hydroshare_1    | TypeError: 'type' object has no attribute '__getitem__'